http_check/datadog_checks/http_check/config.py

```diff
@@ -41,8 +41,8 @@ def from_instance(instance, default_ca_certs=None):
     method = instance.get('method', 'get')
     data = instance.get('data', {})
     tags = instance.get('tags', [])
-    client_cert = instance.get('client_cert')
-    client_key = instance.get('client_key')
+    client_cert = instance.get('tls_cert') or instance.get('client_cert')
+    client_key = instance.get('tls_private_key') or instance.get('client_key')
```

Comment:
Wouldn't this be covered in the remapper? https://github.com/DataDog/integrations-core/blob/master/http_check/datadog_checks/http_check/http_check.py#L42-L43

Reply:
I see. In that case do you need to handle the very rare chance that they use both `tls_cert` and `client_cert`?

Reply:
It's handled like in the requests wrapper. If both are set, `tls_cert` takes precedence, then `client_cert`.